src/main/java/org/jfrog/hudson/maven3/ArtifactoryMaven3Configurator.java

```diff
@@ -394,7 +395,13 @@ public void buildEnvVars(Map<String, String> env) {
             public boolean tearDown(AbstractBuild build, BuildListener listener) {
                 Result result = build.getResult();
                 if (deployBuildInfo && result != null && result.isBetterOrEqualTo(Result.SUCCESS)) {
-                    String buildName = BuildUniqueIdentifierHelper.getBuildNameConsiderOverride(ArtifactoryMaven3Configurator.this, build);
+                    EnvVars env;
+                    try {
+                        env = build.getEnvironment(listener);
+                    } catch (Exception e) {
+                        env = null;
+                    }
```

Comment:
It looks like here you chose to wrap the ```build.getEnvironment```  call with try catch, unlike in other places where you used this API.
Why is that?